qdrant-landing/content/documentation/quick-start.md

```diff
@@ -4,47 +4,28 @@ weight: 11
 aliases:
   - quick_start
 ---
-
 # Quickstart
 
-## Installation
-
-The easiest way to use Qdrant is to run a pre-built image. To do this, make sure Docker is installed on your system.
+## Recommended Workflow
 
-Download image from [DockerHub](https://hub.docker.com/r/qdrant/qdrant):
+There are two ways of using Qdrant locally: [1) Local Mode](#option-1-local-mode) and [2) Docker Mode](#option-2-docker-mode). If you are a Python developer, we recommend that you first try Local Mode in [Qdrant Client](https://github.com/qdrant/qdrant-client), as it only takes a few moments to get setup. Then, you may further experiment with Qdrant Docker containers. When you are more comfortable with Qdrant, then you should deploy your app to a Free Tier [Qdrant Cloud](../cloud/quickstart-cloud/) cluster.
```

Comment:
Again, I don't like Local Mode as first. We only really show it with Python . You can state in the docker instructions, or in the beginning, that if you will be working with Python and prototyping or debugging, then Local Mode may be simpler for you.